src/analysis/convolution.lean

```diff
@@ -1538,3 +1539,53 @@ lemma has_compact_support.cont_diff_convolution_left [μ.is_add_left_invariant]
 by { rw [← convolution_flip], exact hcf.cont_diff_convolution_right L.flip hg hf }
 
 end with_param
+
+section nonneg
+
+variables {μ ν : measure ℝ} [sigma_finite μ] [sigma_finite ν]
+  [is_add_right_invariant μ] [has_no_atoms ν]
+  [normed_space ℝ E] [normed_space ℝ E'] [normed_space ℝ F]
+  [complete_space E] [complete_space E'] [complete_space F]
+
+/-- Variant of `integral_convolution` for integrals over the non-negative reals. -/
```

Comment:
Can you rather explain in the docstring what the lemma is saying? Typically, this text will show up as a mouseover text in a different file, for a user who hasn't a clue what `integral_convolution` is about.